<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameWindow</class>
 <widget class="QMainWindow" name="GameWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Doppelkopf Game</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#line, QWidget#line_2, QWidget#line_3 {
	color: rgb(108,108,108);
	width: 1px;
}
QWidget#statusBar {
background: url(:/statusbar/dark-background.png) repeat-x;
}
QTableCornerButton::section {
border: none;
background: rgb(55,55,55);
}
QLabel#labelLength {
color:white;
}
 QLabel#labelClock {
color:white;
font-size: 19px;
}
QToolButton#toolButtonState, QToolButton#toolButtonToggleSidebar,  QToolButton#toolButtonAddRound, QToolButton#toolButtonAddDrinks, QToolButton#toolButtonAddSchmeisserei, QToolButton#toolButtonSetComment, QToolButton#toolButtonStop, QToolButton#toolButtonSettings{
border: none;
background: none;
}
QSplitter:handle {
background-color: rgb(108,108,108);
}
QToolButton#pushButtonAddPlayers {
	color: white;
	border: 1px solid rgb(108,108,108);
	border-left: none;
	border-bottom: none;
	height: 32px;
	font-size: 14px;
}
PlayersListWidget {
	background-color: rgba(0,0,0,0);
	border:none;
	border-right: 1px solid rgb(108,108,108);
}
QCheckBox {
	color: white;
}
QLabel {
	color: white;
}
QWidget#widgetCreateGame {
	border-bottom: 1px solid rgb(108,108,108);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>1</number>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,1">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,1">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QTableView" name="tableViewInformation">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>40</number>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="Line" name="line_3">
            <property name="frameShadow">
             <enum>QFrame::Plain</enum>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableView" name="tableViewOverview">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="autoFillBackground">
             <bool>true</bool>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="verticalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="horizontalScrollBarPolicy">
             <enum>Qt::ScrollBarAlwaysOff</enum>
            </property>
            <property name="autoScroll">
             <bool>false</bool>
            </property>
            <property name="verticalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="horizontalScrollMode">
             <enum>QAbstractItemView::ScrollPerPixel</enum>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <property name="cornerButtonEnabled">
             <bool>false</bool>
            </property>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>40</number>
            </attribute>
            <attribute name="verticalHeaderVisible">
             <bool>false</bool>
            </attribute>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QWidget" name="widgetCreateGame" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>24</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="Line" name="line_2">
             <property name="frameShadow">
              <enum>QFrame::Plain</enum>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="0,1">
             <property name="rightMargin">
              <number>0</number>
             </property>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <property name="spacing">
                <number>0</number>
               </property>
               <item>
                <widget class="PlayersListWidget" name="listWidgetPlayers">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="minimumSize">
                  <size>
                   <width>250</width>
                   <height>0</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>250</width>
                   <height>16777215</height>
                  </size>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::NoFrame</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QToolButton" name="pushButtonAddPlayers">
                 <property name="sizePolicy">
                  <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                   <horstretch>0</horstretch>
                   <verstretch>0</verstretch>
                  </sizepolicy>
                 </property>
                 <property name="text">
                  <string>Add Players</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="rightMargin">
                <number>12</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <layout class="QFormLayout" name="formLayout">
                 <property name="horizontalSpacing">
                  <number>6</number>
                 </property>
                 <property name="verticalSpacing">
                  <number>12</number>
                 </property>
                 <property name="leftMargin">
                  <number>12</number>
                 </property>
                 <property name="topMargin">
                  <number>12</number>
                 </property>
                 <property name="rightMargin">
                  <number>12</number>
                 </property>
                 <property name="bottomMargin">
                  <number>12</number>
                 </property>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>Site:</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="PlacesComboBox" name="comboBoxSite"/>
                 </item>
                 <item row="1" column="0" colspan="2">
                  <widget class="QCheckBox" name="checkBoxMitPflichtSoli">
                   <property name="text">
                    <string>Mit Pflichtsoli</string>
                   </property>
                   <property name="checked">
                    <bool>true</bool>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item>
                <widget class="QDialogButtonBox" name="buttonBox">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="standardButtons">
                  <set>QDialogButtonBox::Ok</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="styleSheet">
           <string notr="true">QWidget {
	color: rgb(108,108,108);
}</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Plain</enum>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="ScrollArea" name="scrollAreaGraph">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="GraphAxis" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>668</width>
             <height>93</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="GraphAxis" name="graphAxis" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="GameLengthWidget" name="gameLengthWidget" native="true"/>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="GameGraphWidget" name="graphWidget" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout"/>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="ShadowFrame" name="widgetStatisticsFrame" native="true">
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="statusBar" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>52</width>
        <height>52</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>52</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>12</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="labelLength">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonAddRound">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resource/statusbar/statusbar.qrc">
           <normaloff>:/statusbar/newround.png</normaloff>:/statusbar/newround.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonIconOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonAddSchmeisserei">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resource/statusbar/statusbar.qrc">
           <normaloff>:/statusbar/schmeisserei.png</normaloff>:/statusbar/schmeisserei.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonAddDrinks">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resource/statusbar/statusbar.qrc">
           <normaloff>:/statusbar/drink.png</normaloff>:/statusbar/drink.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonSetComment">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="../../resource/statusbar/statusbar.qrc">
           <normaloff>:/statusbar/comment.png</normaloff>:/statusbar/comment.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="toolButtonToggleSidebar">
         <property name="minimumSize">
          <size>
           <width>17</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>17</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="iconSize">
          <size>
           <width>17</width>
           <height>16</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextOnly</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelClock">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionPlayPause">
   <property name="icon">
    <iconset resource="../../resource/statusbar/statusbar.qrc">
     <normaloff>:/statusbar/pause.png</normaloff>:/statusbar/pause.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionAdd_round">
   <property name="icon">
    <iconset resource="../../resource/statusbar/statusbar.qrc">
     <normaloff>:/statusbar/newround.png</normaloff>:/statusbar/newround.png</iconset>
   </property>
   <property name="text">
    <string>Add round...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAdd_schmeisserei">
   <property name="icon">
    <iconset resource="../../resource/statusbar/statusbar.qrc">
     <normaloff>:/statusbar/schmeisserei.png</normaloff>:/statusbar/schmeisserei.png</iconset>
   </property>
   <property name="text">
    <string>Add schmeisserei...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionToggleSidebar">
   <property name="icon">
    <iconset resource="../../resource/statusbar/statusbar.qrc">
     <normaloff>:/statusbar/sidebar-left.png</normaloff>:/statusbar/sidebar-left.png</iconset>
   </property>
   <property name="text">
    <string>Hide sidebar</string>
   </property>
  </action>
  <action name="actionAdd_drinks">
   <property name="icon">
    <iconset resource="../../resource/statusbar/statusbar.qrc">
     <normaloff>:/statusbar/drink.png</normaloff>:/statusbar/drink.png</iconset>
   </property>
   <property name="text">
    <string>Add drinks...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionStats">
   <property name="text">
    <string>Stats</string>
   </property>
  </action>
  <action name="actionStop_Game">
   <property name="icon">
    <iconset resource="../../resource/statusbar/statusbar.qrc">
     <normaloff>:/statusbar/stop.png</normaloff>:/statusbar/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop Game</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../../resource/statusbar/statusbar.qrc">
     <normaloff>:/statusbar/settings.png</normaloff>:/statusbar/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCheck_for_updates">
   <property name="text">
    <string>Check for updates...</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrollArea</class>
   <extends>QScrollArea</extends>
   <header location="global">ui/widgets/scrollarea.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GraphAxis</class>
   <extends>QWidget</extends>
   <header location="global">ui/game/graphaxis.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GameGraphWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/game/gamegraphwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ShadowFrame</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/shadowframe.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PlayersListWidget</class>
   <extends>QListWidget</extends>
   <header location="global">ui/widgets/playerslistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>PlacesComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">ui/widgets/placescombobox.h</header>
  </customwidget>
  <customwidget>
   <class>GameLengthWidget</class>
   <extends>QWidget</extends>
   <header>ui/game/gamelengthwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resource/statusbar/statusbar.qrc"/>
 </resources>
 <connections/>
</ui>
